code/lang/TTTLanguage.cs

```diff
@@ -62,23 +57,40 @@ public static void LoadLanguages()
 
         public static Language GetLanguageByCode(string name)
         {
-            if (!Languages.TryGetValue(name, out Language lang))
+            Language lang = null;
+
+            try
             {
-                Log.Warning($"Tried to get a language that does not exist: '{name}'.");
+                if (!Languages.TryGetValue(name, out lang))
+                {
+                    Log.Warning($"Tried to get a language that does not exist: '{name}'.");
+                }
             }
+            catch (Exception) { }
```

Comment:
Does this throw an exception? I thought it just made `lang` null (but I can't remember if that was javascript or C#)

Reply:
Yea, if  `Languages` or `name` is null